response_operations_ui/views/respondents.py

```diff
@@ -447,5 +438,5 @@ def _generate_pagination_href(email_address="", first_name="", last_name=""):
     if last_name != "":
         href_string_list.append("lastname=" + last_name)
     # This is needed for custom hrefs as per the flask-paginate docs
-    href_string_list.append("page={0}")
+    href_string_list.append("page=")
```

Comment:
is this still needed? Should probably remove the reference to flask-paginate either way to avoid confusion

Reply:
Absolutely. I missed this. I'm going to take a look now to see what can be improved.